string model_name
string exposures
---
bool success
